main.c

```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
 		printf("Usage: %s n\n", argv[0]);
 		return 1;
 	}
+	if(argv[1]<0){
```

Comment:
A space between the ")" and "{"

https://bytesclub.github.io/docs/coding_style.pdf